cmd/minikube/cmd/service.go

```diff
@@ -72,20 +72,22 @@ var serviceCmd = &cobra.Command{
 			cmdutil.MaybeReportErrorAndExit(err)
 		}
 
-		url, err := cluster.GetServiceURL(api, namespace, service, serviceURLTemplate)
+		urls, err := cluster.GetServiceURLs(api, namespace, service, serviceURLTemplate)
```

Comment:
Could you change the help text a bit to clarify that the template is applied to each URL in the case of multiple? And that we'll print them one line at a time.